src/main/resources/scripts/tpcds/data_maintenance_dependent/DF_CS.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
```

Comment:
Nit. Maybe it's only to me, but it seems more intuitive to split this file into two files, one for each statement, since the `DELETE` is executed in a loop and this gives the impression it's a sequential SQL script. WDYT?

Reply:
Makes sense.